a105_lidar:
  ros__parameters:
    port: "/dev/ttyUSB0"
    baud: 230400
    frame_id: "laser"
    inc_deg: 0.5
    range_min: 0.05
    range_max: 8.0
    angle_offset_deg: 0.0
    clockwise: false
